airbase/pom.xml

```diff
@@ -416,6 +419,7 @@
                         <showDeprecation>true</showDeprecation>
                         <showWarnings>true</showWarnings>
                         <parameters>true</parameters>
+                        <enablePreview>${air.compiler.enable-preview}</enablePreview>
```

Comment:
Is it possible to automatically add the surefire argline when this is enabled?

Reply:
Let me see if we can do that with a profile